tests/unit/adaptors/graph/test_chains.py

```diff
@@ -327,27 +432,45 @@ def test_chains_that_reenter_imported_package_dont_stop_longer_chains_being_incl
             ),
         }
 
-    def test_doesnt_lose_import_details(self):
+    @pytest.mark.parametrize(
+        "as_packages",
+        (
+            False,
+            True,
```

Comment:
Could we remove any necessary commas here so the `parametrize` decorator all gets formatted onto one line?